Actors/VisualObjects/VisualObjects/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -1,35 +1,28 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="VisualObjectsApplication" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
-   <Parameters>
-      <Parameter Name="WebServiceInstanceCount" DefaultValue="1" />
-      <Parameter Name="VisualObjects.ActorService_PartitionCount" DefaultValue="9" />
-      <Parameter Name="VisualObjects.ObjectCount" DefaultValue="9" />
-   </Parameters>
-   <ServiceManifestImport>
-      <ServiceManifestRef ServiceManifestName="VisualObjects.ActorServicePkg" ServiceManifestVersion="1.0.0" />
-   </ServiceManifestImport>
-   <ServiceManifestImport>
-      <ServiceManifestRef ServiceManifestName="VisualObjects.WebServicePkg" ServiceManifestVersion="1.0.0" />
-      <ConfigOverrides>
-         <ConfigOverride Name="Config">
-            <Settings>
-               <Section Name="VisualObjectsBoxSettings">
-                  <Parameter Name="ObjectCount" Value="[VisualObjects.ObjectCount]" />
-               </Section>
-            </Settings>
-         </ConfigOverride>
-      </ConfigOverrides>
-   </ServiceManifestImport>
-   <DefaultServices>
-      <Service Name="VisualObjects.ActorService">
-         <StatefulService ServiceTypeName="VisualObjects.ActorServiceType" TargetReplicaSetSize="3" MinReplicaSetSize="2">
-            <UniformInt64Partition PartitionCount="[VisualObjects.ActorService_PartitionCount]" LowKey="-9223372036854775808" HighKey="9223372036854775807" />
-         </StatefulService>
-      </Service>
-      <Service Name="VisualObjects.WebService">
-         <StatelessService ServiceTypeName="VisualObjects.WebServiceType" InstanceCount="[WebServiceInstanceCount]">
-            <SingletonPartition />
-         </StatelessService>
-      </Service>
-   </DefaultServices>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="VisualObjectsApplicationType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
+   <Parameters>
+      <Parameter Name="VisualObjects.WebService_InstanceCount" DefaultValue="1" />
+      <Parameter Name="VisualObjects.ActorService_PartitionCount" DefaultValue="10" />
+      <Parameter Name="VisualObjects.ActorService_MinReplicaSetSize" DefaultValue="2" />
```

Comment:
Can we fix it here because it is wrong? We should also fix the template to set it to 3 & 3 then